Python/Product/VSCode/AnalysisVsc/LanguageServer.cs

```diff
@@ -153,6 +154,9 @@ private void OnUnregisterCapability(object sender, UnregisterCapabilityEventArgs
         #region Workspace
         [JsonRpcMethod("workspace/didChangeConfiguration")]
         public async Task DidChangeConfiguration(JToken token, CancellationToken cancellationToken) {
+            // Change configuration comes right after the initialization
+            WaitForServerInitComplete(cancellationToken);
```

Comment:
Why `await server.CompleteInitialization` isn't enough?

Reply:
It yields, we get next request. So we end up receiving requests without having initialization complete. One option to add await CompleteInitialization to sync context and call it before any request.

Reply:
But `DidChangeConfiguration` is called through sync context already. We have it in JsonRPC, right?

Reply:
It called _sequentially_ but I do see the yield. Perhaps our sync context is incorrect.